java/src/main/kotlin/io/spine/validation/java/InTimeGenerator.kt

```diff
@@ -101,14 +101,14 @@ private class InSpineTimeGenerator(
     inTime: InTime,
     ctx: GenerationContext
 ) : SimpleRuleGenerator(ctx) {
+
     private val time = inTime.time
-    override fun condition(): Expression {
-        val compareTo = MethodCall(
+
+    override fun condition(): Expression<Boolean> =
+        MethodCall(
             ctx.fieldOrElement!!,
             time.temporalMethod()
         )
-        return time.formatJavaComparison(compareTo)
```

Comment:
A call to `time.temporalMethod()` returns `boolean`. I don't know how `time.formatJavaComparison()` is applicable here. I've removed it.